io.openliberty.java.internal_fat_22/README.md

```diff
@@ -0,0 +1,89 @@
+# Repo for quick testing of Liberty apps
+Especially useful for new Java versions
+
+### For versions of Java that already have gradle support
+
+To build the WAR file locally, from the root directory of the open-liberty-misc repository run:
+
+```
+./gradlew io.openliberty.java.internal_fat_22:build
+```
+
+To build+run locally, this currently requires some manual effort:
+* Note, the **build.gradle** file is configured to look for a Liberty install under `/wlp`. You might need to change it to point it to your local test Liberty install instead.
+* Copy the `jvm.options` file and the contents of the `server.env` file from the project's `run` directory and add them to your `/wlp/usr/servers/io.openliberty.java.internal_fat_22Server/server.env` file (or equivalent location).
+* Change to the root directory of this local `io.openliberty.java.internal_fat_22` repository and run:
+
+```
+./gradlew io.openliberty.java.internal_fat_22:start
+```
+
+---
+
+### For versions of Java that do not have gradle support yet
```

Comment:
Everything below this line seems like general project information... should this be moved to the base README.md file?  I could also see this going into the DEVELOPER.md file at the root of this project to keep information that developers would need to know when a new version of java comes out.

Reply:
I don't want to move the README.md information in this file to the root of `open-liberty-misc` since this repo was designed with the intent that other components could put misc code in this repo as well.

Instead, I have cleaned up the README.md file to remove some of its verbosity.